tests/Unit/assets/stubs/mysql.stub

```diff
@@ -1,5 +1,5 @@
   mariadb:
-    image: amazeeio/mariadb
+    image: uselagoon/mariadb
```

Comment:
```suggestion
    image: uselagoon/mariadb-10.11
```

Reply:
I thought it was weird that the mysql stub was referencing mariadb

Reply:
historically it had to, until we made MySQL images :partying_face: (and in non-drupal world, the creds are the same)